src/main/resources/stepsToRun.config

```diff
@@ -1,16 +1,14 @@
 # This file is used to specify which steps in DownloadDirectory to execute.
 # Comment out any steps that don't need to be run.
-protegeexporter
+
 #DatabaseDumps
 #BioPAX2
 #BioPAX3
 #GSEAOutput
 #FetchTestReactomeOntologyFiles
 #PathwaySummationMappingFile
```

Comment:
If these are _still_ commented out, should they just be removed? Or at least put a note in explaining why these entries are present but never used.

Reply:
No that was an excellent catch - I will uncomment them (caught this in addlinks but not DD)